docs/sphinx_qtile.py

```diff
@@ -51,6 +51,8 @@
     .. compound::
 
         Supported bar orientations: {{ obj.orientations }}
+
+        Supported backends: {{ ", ".join(obj.supported_backends) }}
```

Comment:
This would show for every widget but we could add a condition so it's only shown if the widget has a different value for `supported_backends`.

Reply:
Should also remove the text in the widget docstring if we keep this.

The advantage of having sphinx do it is that widgets only need to add the `supported_backends` value and the rest is automatic.

Reply:
Yeah I think it would be nice to avoid the noise of having it displayed for all widgets. Is it straight forward to do with sphinx?

Reply:
It's already done.

Reply:
Ah, I re-read like 3 times trying to figure out, completely missed line 146. Trying out the new "high contrast" github dark mode and ironically it has made the background of the green lines very low contrast, lol.